proto/v1alpha1/resource.proto

```diff
@@ -51,12 +51,26 @@ message LabelTerm {
         // Label value is equal.
         EQUAL = 1;
         // Label doesn't exist.
-        NOT_EXISTS = 2;
+        NOT_EXISTS = 2 [deprecated=true];
```

Comment:
why are we removing it? I don't think it can be expressed via anything else?

Reply:
ah, got it, we have `EXISTS`, but I feel like it's a breaking change

Reply:
not breaking if we keep it but mark as deprecated.